index.html

```diff
@@ -0,0 +1,138 @@
+          <input type="checkbox" id="nav-toggle" />
+          <label class="nav-alter" for="nav-toggle">
+            <span></span>
+            <span></span>
+            <span></span>
+          </label>
+          <!--end hamburger bar-->
+    <div class="container">
+        <!--start navigation bar-->
+        <nav class="menu">
+          <ul>
+            <li><a href="#home">Home</a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#contact">Contact</a></li>
+            <li><a href="#signup">Sign up</a></li>
+          </ul>
+        </nav>
+        <!--end navigation bar-->
+
+      <!--start hero image-->
+      <!--<div class="hero">
+        <div class="hero__text">
+          <h1>Join bootcamp!</h1>
+          <button>Sign up</button>
+        </div>
+      </div>-->
+      <!--end hero-->
+
+      <!-- start hero video-->
+      <header class="video-header">
+        <video autoplay loop muted poster="img/business-web.png">
+          <source src="videos/heroVideo.mp4" type="video/mp4" />
+          Your browser doesn't support the video tag. The sadness :(
+        </video>
+        <div class="hero__text">
+          <h1>Join Bootcamp!</h1>
+          <button><a href="#signup">Sign up</a></button>
+        </div>
+      </header>
+      <!-- end hero video-->
+
+      <div class="mainSection">
+        <!-- Signup form -->
+        <section class="signupForm">
+          <h1>Signup</h1>
+          <form class="signupForm__info" action="https://httpbin.org/anything" method="post">
+
+            <input  type="text" 
+                    id="fname" 
+                    name="fname" required 
+                    minlength="4" 
+                    maxlength="15" 
+                    placeholder="First name"
+                    autofocus="autofocus" /><br><br>
+
+            <input  type="text" 
+                    id="lname" 
+                    name="lname" required 
+                    minlength="4" 
+                    maxlength="15"
+                    placeholder="Last name" /><br><br>
+
+            <fieldset>
+              <legend>Gender</legend>
+              <label class="containerRadio">Male
+                <input type="radio" name="radio">
+                <span class="checkmarkRadio"></span>
+              </label>
+              <label class="containerRadio">Female
+                <input type="radio" name="radio">
+                <span class="checkmarkRadio"></span>
+              </label>
+            </fieldset><br><br>
+
+            <input type="email" id="email" name="email" required placeholder="Email"><br><br>
+
+            <label class="containerCheckbox" for="term"><i>Agree to term and conditions</i>
+              <input type="checkbox" id="term" name="term" value="Term&condition">
+              <span id="checkmark" class="checkmark"></span>
+            </label><br><br>
+
+            <input id="submit" type="submit" value="Submit" disabled>
```

Comment:
When I push the submit-button I get to the httpbin-site. Could be an option to use a "preventDefault"-function to skip that step?

Reply:
I am not clear what are you talking about.
Is it  <form class="signupForm__info" action="https://httpbin.org/anything" method="post">
Instead of  action="https://httpbin.org/anything" to use "preventDefault"